### DISCUSSION TITLE
Dịch instance segmentation, semantic segmentation

### DISCUSSION THREAD

**Discussion by minhduc0711** [mlbvn/handson-ml2-vn#35] [Terminology] [answered]
Từ `semantic segmentation` trước mình dịch là `phân vùng theo ngữ nghĩa`, nhưng giờ nhiều người thấy không phù hợp nên mình bàn lại từ này. Mình tạm đề xuất các option sau:
* `semantic segmentation` -> `phân vùng theo nhóm`, `theo ý nghĩa`?
* `instance segmentation` -> `phân vùng theo vật thể`, `theo thực thể`?

Mọi người cho ý kiến nhé @mlbvn/handson-ml2

**Comment by ngcthuong** [comment]
Instance learning mình dịch là "học theo mẫu" rồi thì mình dịch instance segmentation thành "phân vùng theo mẫu" cũng được.

**Reply by quangvnai** [reply]
* `instance` trong hai cụm này khác nhau mà anh nhỉ? Em vote `phân vùng theo vật thể`.

**Comment by quangvnai** [comment]
* Mình vote `semantic segmentation` dịch thành `phân vùng theo nhóm/lớp/hạng mục` hơn là `theo ý nghĩa`.

**Comment by lkhphuc** [comment]
`semantic segmentation`: `phân vùng theo loại` được không
Còn thực thể với vật thể cũng không khác gì nhau lắm nên cứ giữ nguyên như hiện giờ nhỉ?

**Reply by minhduc0711** [reply]
ừm mình thấy ok

**Reply by minhduc0711** [reply]
hmm nghĩ lại thì `vật thể` nghe có vẻ quen thuộc hơn so với `thực thể`

**Comment by minhduc0711** [comment, marked as answer]
Thôi cứ tạm chốt thế này đã nhé
* `instance segmentation`: `phân vùng theo vật thể`
* `semantic segmentation`: `phân vùng theo nhóm`
